src/sentry/static/sentry/app/utils/cookies.jsx

```diff
@@ -0,0 +1,17 @@
+export default {
```

Comment:
I split this out so that the `getCookie` function could be reused. It was only used inside `index.js` before. Now it's just exposed to be able to do `cookies.get(name)`.